### ISSUE TITLE
is hadoop Necessary？

### ISSUE THREAD

**Issue by keasonsun** [NGDATA/hbase-indexer#111]
can i use hbase-indexer without hadoop ?  I  just want to index Hbase'data to Solr.

**Comment by saintstack** [comment]
HBase runs on Hadoop HDFS.

**Comment by keasonsun** [comment]
> HBase runs on Hadoop HDFS.

Hbase can run without Hadoop.
Why we need Hadoop?

**Comment by keasonsun** [comment]
I had done it.  
it's alright to use hbase-indexer without hadoop.
My problem is that there is no host mapping of hbase-indexer's machine in hbase'machine /etc/hosts.

**Comment by saintstack** [comment]
This is wrong place for this discussion. Please ask on mailing list. To learn how hbase uses hadoop, please see the refguide.  Read the few paragraphs starting here: http://hbase.apache.org/book.html#standalone_dist . It describes how Hadoop HDFS is needed running in distributed mode.
